src/integration-tests/src/secret_manager/openapi_locational.rs

```diff
@@ -212,11 +189,12 @@ async fn run_secret_versions(
 
     println!("\nTesting get_secret_version_by_project_and_location_and_secret_and_version()");
     let get = client
-        .get_secret_version_by_project_and_location_and_secret_and_version()
-        .set_project(project_id)
-        .set_location(location_id)
-        .set_secret(secret_id)
-        .set_version(version_id)
+        .get_secret_version_by_project_and_location_and_secret_and_version(
```

Comment:
nice, I like this a lot better!

Reply:
Most client methods are fine.  It is less than ideal that some require fields are not required by the method...